gui.js

```diff
@@ -1552,9 +1552,24 @@ IDE_Morph.prototype.fixLayout = function (situation) {
 IDE_Morph.prototype.setProjectName = function (string) {
     this.projectName = string.replace(/['"]/g, ''); // filter quotation marks
     this.hasChangedMedia = true;
+    this.updateDocumentTitleText();
     this.controlBar.updateLabel();
 };
 
+IDE_Morph.prototype.updateDocumentTitleText = function (string) {
+    var text = this.getDocumentTitleText(string || this.projectName);
+    title = document.getElementsByTagName('title')[0];
```

Comment:
okay, I admittedly suck at web programming, so I don't know better. Can you explain to me why we can't just write

```JS
    document.title = 'Snap! - ' + this.projectName;
```

(?) Thanks!